test/Microsoft.TestPlatform.SmokeTests/DataCollectorTests.Coverlet.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.SmokeTests
+{
+    using Microsoft.TestPlatform.TestUtilities;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+
+    [TestClass]
+    public class DataCollectorTestsCoverlets : IntegrationTestBase
+    {
+        [TestMethod]
+        public void RunCoverletCoverage()
+        {
+            string projectFileName = this.GetProjectAssetFullPath("CoverletCoverageTestProject.csproj", "CoverletCoverageTestProject.csproj");
```

Comment:
I am confused why is this doubled, please comment.

Reply:
It's a tmp typo...this PR is WIP I've updated on my local branch with `string projectFileName = this.GetProjectFullPath("CoverletCoverageTestProject.csproj");` but I'll wait to push until test work...I need an help to undestand if my issue is something already saw https://github.com/microsoft/vstest/pull/2330#issuecomment-586622410